src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -50,6 +51,11 @@ public AdbRunner(ILogger log, string adbExePath = "")
             }
         }
 
+        public void SetActiveDevice(string deviceSerialNumber)
+        {
+            _currentDevice = deviceSerialNumber;

```

Comment:
check for null?

Reply:
Null is fine, the only place where this is used always calls string.IsNullOrEmpty.